### PR TITLE
added unit tests for OcHiddenAnnouncer component

### LINKED ISSUE #1366: Add unit tests to first batch of components
Add unit tests for the following components:
- [x] OcAvatar.vue (#1424)
- [x] OcAvatarLink.vue (#1430)
- [x] OcAccordionItem.vue (#1381)
- [x] OcAccordion.vue (#1389)
- [x] OcActionDrop.vue (#1394)
- [x] OcAlert.vue (#709)
- ~OcAppBar.vue~ (currently unused and out of scope for this ticket)
- ~OcAppLayout.vue~ (currently unused and out of scope for this ticket)
- [x] OcAppSideBar.vue (#1410)
- [x] OcAutocomplete.vue (#1440)
- [x] OcBreadcrumb.vue (#806)
- [x] OcButton.vue (#1364)
- [x] OcCheckbox.vue (#1408)
- [X] OcDatepicker.vue~ (#1661)
- ~OcDisclosureDrop.vue~ (currently unused and out of scope for this ticket)
- [x] OcDrop.vue (#1269)
- [x] OcDropzone.vue (#1439)
- [x] OcGrid.vue (#1441)
- [x] OcHiddenAnnouncer.vue (#1445)
- [x] OcIcon.vue (#1407)
- [x] OcImage (#1446)
- ~OcList.vue~ (currently unused and out of scope for this ticket)
- [x] OcLoader.vue (#1449)
- [x] OcLogo.vue (#1450)
- [x] OcModal.vue (#723)
- [x] _OcNavbarItem.vue (#1451)
- [x] OcNavbar.vue (#1452)
- [x] OcNavItem.vue (#1475)
- [x] OcNav.vue (#1474)
- [x] OcNotificationMessage.vue (#1476)
- [x] OcNotifications.vue (#1489)
- ~OcProgressPie.vue~ (currently unused and out of scope for this ticket)
- [x] OcProgress.vue (#819)
- [x] OcRadio.vue (#1490)
- [x] OcSearchBar.vue (#1495)
- [ ] OcSelect.vue
- ~_OcSidebarNavDivider.vue~ (deprecated component)
- ~_OcSidebarNavHeader.vue~ (deprecated component)
- [ ] OcSidebarNavItem.vue
- [x] OcSidebarNav.vue (#1300)
- [ ] OcSpinner.vue
- [ ] OcStatusIndicators.vue
- ~OcSwitch.vue~ (currently unused and out of scope for this ticket)
- ~OcTabbedPanel.vue~ (currently unused and out of scope for this ticket)
- ~OcTabbedTab.vue~ (currently unused and out of scope for this ticket)
- ~OcTabbed.vue~ (currently unused and out of scope for this ticket)
- ~OcTabItem.vue~ (currently unused and out of scope for this ticket)
- ~OcTabs.vue~ (currently unused and out of scope for this ticket)
- [x] OcTag.vue (#927)
- [x] OcTextarea.vue (#1517 )
- [x] OcTextInput.vue (#1329, #1498)
- ~_OcTopBarItem.vue~ (deprecated component)
- ~_OcTopBarLogo.vue~ (deprecated component)
- ~OcTopBar.vue~ (deprecated component)
- ~OcUser.vue~ (deprecated component)

### Motivation
- https://github.com/owncloud/web/issues/5217

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
- added unit tests for `OcHiddenAnnouncer` component

## Related Issue

- Fixes #1366 

## How Has This Been Tested?

- 🤖 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [x] Code changes
- [x] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation added/updated

## Open tasks:

- [ ] ...

### CODE REVIEW

File: src/components/OcHiddenAnnouncer.vue [hunk 1]
```diff
@@ -33,14 +33,16 @@ export default {
       type: String,
       required: false,
       default: "polite",
+      validator: value => {
+        return value.match(/(polite|assertive|off)/)
```
Comment (kiranparajuli589) [inline src/components/OcHiddenAnnouncer.vue:37]: as per the prop docs, only these values should work for `aria-live`.

File: src/components/OcHiddenAnnouncer.vue [hunk 2]
```diff
@@ -33,14 +33,16 @@ export default {
...
+      },
     },
     /**
      * The announcement text itself.
      **/
     announcement: {
       type: String,
       required: true,
-      default: "",
```
Comment (kiranparajuli589) [inline src/components/OcHiddenAnnouncer.vue:43]: no default is needed for required prop

**Review by individual-it — COMMENTED** [review]
LGTM, @LukasHirt @kulmann are you happy with the code changes?

**Review by LukasHirt — APPROVED** [review]
LGTM

**Pull request merged**